localcoinbot.py

```diff
@@ -31,21 +41,23 @@ def good_permissions():
         can_invite_users=True,
     )
 
+
 def bad_permissions():
     return ChatPermissions(None)
 
 
 def start(update, context):
     if is_public_chat(update, context):
+        remove_command(context, update)
         bot_hello = context.bot.send_message(
-            chat_id=update.message.chat.id,
+            chat_id=update.effective_message.chat.id,
```

Comment:
I wonder why they felt the need to do this when they released the new package?

Reply:
Apparently, you can do both ways, but from my understanding, this protects you from edited msgs etc